dtk/process.py

```diff
@@ -513,7 +519,8 @@ def nextpow2(i):
         L = data.shape[0]  # length of data if (n,)
     # calculate the closest power of 2 for the length of the data
     n = nextpow2(L)
-    Y = fft(data, n)/L  # divide by L for scaling
+
+    Y = fft(data, n, norm="forward") 
```

Comment:
Does this do the same thing as the prior `/L`? Or does it change the result?

Reply:
> The power functions change the default "forward" policy (which is equal to scaling by 1/N, as previously done), to "ortho" to ensure that Parseval's theorem is satisfied.

Ok, so the same.

Reply:
So if no normalization occurred pre-numpy 1.10, I wonder if my implementation assumed that. The docs now say that the default is backwards, so I guess this code normalized the already backwards normalized result.

Reply:
To be an invertible transformation ( $x(n) = \mathrm{IDFT}(\mathrm{DFT}(x(n))$ ), the normalization factor $\frac{1}{N}$ can be assigned to both the forward and the backward transformation.

It is my impression that the backwards case where the normalization happens in the inverse DTF is more common, but I could be wrong. 

The old numpy did no normalization of the "forward" DFT ($X(k) = \mathrm{DFT}(x(n))$) and normalized the inverse DFT. This is exactly the default "backwards" case now. So old and new have the same default behaviour. Your code did the normalization of the forward DFT manually, effectively switching to the "forward" convention. For the spectrum, this is fine and there are good reasons to do the forward normalization. I guess the most important one is that the amplitudes are independent of the signal length. An error may only occur if someone does an default inverse DFT which would apply the normalization again (backwards).

Reply:
Glad to have an electrical engineering looking at this!